app/Models/Event.php

```diff
@@ -59,27 +59,27 @@ class Event extends Model
 
     protected $guarded = [];
 
-    public function user(){
+    public function user() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->